### PR TITLE
Feature/add 18

### PR DESCRIPTION
**Pull request by zapjelly**
I'll fix the merge conflicts after you have eyeballed the changes, sorry it's a lot.

This includes the merge from Annex, auto update and code signing work and client feedbacks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,33 @@
...
+
 ```
 
-2. Run the following:
+2. Run the following (require Mac):
 
 ```
-yarn install
-yarn build
+yarn
```
Comment (cchew) [inline README.md:23]: Is this step meant to be `yarn install` rather than just `yarn`?

File: public/autoUpdate.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ipcRenderer.on("update_available", () => {
+	ipcRenderer.removeAllListeners("update_available")
+	message.innerText = "A new update is available. Downloading now..."
+	notification.classList.remove("hidden")
+})
+
+ipcRenderer.on("update_downloaded", () => {
+	ipcRenderer.removeAllListeners("update_downloaded")
+	message.innerText =
+		"Update Downloaded. It will be installed on restart. Restart now?"
```
Comment (cchew) [inline public/autoUpdate.js:38]: Really minor, lowercase d for 'downloaded' please =)

File: public/dev-app-update.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+owner: pragmapartners
```
Comment (cchew) [inline public/dev-app-update.yml:1]: I noticed there is also a copy of this file at the root level - are both needed?

File: src/components/Navbar/Public/PublicNavbar.tsx [hunk 4]
```diff
@@ -23,7 +65,47 @@ const PublicNavbar = ({ hasClients }: IPublicNavbarProps): ReactElement => {
 					<Box className={`${sharedStyle.logoBox} ${sharedStyle.publicLogo}`}>
 						<Logo to={PublicRoutes.Home} />
 					</Box>
-					<Box className={sharedStyle.box}>
+					{/* Only show download on web and show both download options if it doesn't match either */}
+					{!isDesktop && (
```
Comment (cchew) [inline src/components/Navbar/Public/PublicNavbar.tsx:69]: Do we have a conditional path for Linux?

**Review by cchew — COMMENTED** [review]
Looks good!

**Comment by zapjelly** [comment]
merged separately

**Pull request closed without merging**